### ISSUE TITLE
Installer crashes if the package contains files without filename extension

### ISSUE THREAD

**Issue by sebastianhein** [Squirrel/Squirrel.Windows#580]
Hi, 
we are currently evaluating squirrel.windows, it looks very promising so far.

The current setup ist:
I am building the installer on OSX using mono and wine. The Application is written in Unity and exported as a Windows package. Versions are:
- NuGet.CommandLine.3.3.0
- squirrel.windows.1.2.3

I can create a package like that: 

`$ mono NuGet.exe pack test.nuspec -Version 0.1.0 -BasePath ./squirrel_build/ -OutputDirectory ./squirrel_build/`
`$ mono Squirrel-Mono.exe   --releasify squirrel_build/TestApp.0.1.0.nupkg --releaseDir=./squirrel_release`

But when I try to install the created setup.exe on a Windows machine, I get an "installation has failed" message. 

After some analysing we found out that everything works fine for files like "foobar.txt" but crashes if we just put one file into the package that does not contain a filename extension (i.e. "foobar").

Complete log file, aka Das Uberprotokoll: (sorry, parts of it are in german)
[SquirrelSetup.txt](https://github.com/Squirrel/Squirrel.Windows/files/109613/SquirrelSetup.txt)

Thanks,
Sebastian

**Comment by ivanovit** [comment]
We had similar problem but in our case the whole Nuget package was broken. So the fix was just to add file extension. In our case the problematic files were some text files so adding .txt helped.

**Comment by sebastianhein** [comment]
Renaming the files to something.txt is not possible, because we are packaging Unity built apps and these contain files without extension.
Building the installer on windows works, so I suspect the the error is somehow mono related.
We created a little workaround using zipped application bundles that are being unzipped in the installation process. However, we now have installable installers but creating a second installer fails now, because the delta compression fails on osx now. 
I like the squirrel concepts and it seems to work nicely on windows but osx is giving me wrinkles.

**Comment by geary** [comment]
> I am building the installer on OSX using mono and wine.

That's the problem right there.

Honestly you'd be a lot better off using a Windows VM instead of trying to simulate a Windows environment like that. Squirrel is a Windows installer, not a mono+vine-on-OSX installer. 

Since your build works when you run it on Windows, that's what you should do. Just use your favorite virtualization solution to run a genuine Windows VM on your Mac - Parallels, VMware, or VirtualBox. Squirrel builds should work fine on any of them.

That's what I do - I may be developing on Mac hardware, but I do my Windows work in a Parallels VM running Windows 10. Squirrel and all my other Windows apps work great that way.

**Comment by sebastianhein** [comment]
Thanks for the reply.
I'm starting to feel the same way :-)

**Comment by paulcbetts** [comment]
Yeah, the Mono version definitely still has some glitches to work out, but it'd be cool to get it working
